### ISSUE TITLE
Request: add pagination 

### ISSUE THREAD

**Issue by arifje** [HOMMinteractive/hommxmlsitemap#10]
If you have 100k entries, requesting the feed take some time. Adding pagination can solve this I guess, or are there any other options?

**Comment by ammannbe** [comment]
Good idea, also a caching could considered as asked here: https://github.com/HOMMinteractive/hommxmlsitemap/issues/15
Any PR's are welcome.
